content/de/abstraction.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Abstraktion
+status: Feedback erwünscht
+category: Eigenschaft
+---
+
+Mithilfe der Abstraktion werden spezifische Gegebenheiten von IT – Systemen vor dem Nutzer verborgen.
+Dadurch schafft die Abstraktion generische Lösungen, die einfacher zu verstehen sind.
```

Comment:
Ich würde statt "die Abstraktion" eher "...schaffen Abstraktionen" oder "....schafft eine Abstraktion" verwenden.

Reply:
In "...schaffen Abstraktionen" geändert.